src/webhook/mutation/pod_mutator.go

```diff
@@ -794,3 +800,24 @@ func silentErrorResponse(podName string, err error) admission.Response {
 	rsp.Result.Message = fmt.Sprintf("Failed to inject into pod: %s because %s", podName, err.Error())
 	return rsp
 }
+
+// getSecretConfigData fetches the SecretConfig (initSecret.Config) from the pod namespace
+func (m *podMutator) getSecretConfigData(nsName string) (*standalone.SecretConfig, error) {
+	var scs corev1.Secret
+	err := m.apiReader.Get(context.TODO(), client.ObjectKey{Name: dtwebhook.SecretConfigName, Namespace: nsName}, &scs)
+	if err != nil {
+		podLog.Error(err, "Failed to fetch config secret", "name", dtwebhook.SecretConfigName, "namespace", nsName)
```

Comment:
```suggestion
		podLog.Error(err, "failed to fetch config secret", "name", dtwebhook.SecretConfigName, "namespace", nsName)
```

Reply:
Fixed all init capped `podLog.Error` strings in pod_mutator.go.